lib/exp.py

```diff
@@ -30,6 +31,37 @@ def __init__(self, namespace='default'):
         self.k8s = client.CustomObjectsApi()
         self.batch = client.BatchV1Api()
 
+    # Helper methods
+
+    def _retry_poll_api(self, api, max_retries_error, max_retries=30,
+                        retry_interval=1, api_kwargs={}):
+        """
+        Helper function that has a polling loop to retry calling the specified
+        API until it's successful (the client does not throw an Api Exception).
+
+        :param api: Client API to call
```

Comment:
would be useful to elaborate on what's expected here as this is kind of vague.  Is it expecting a string?  some other type of object?  What kind of API's are we talking about here?

Reply:
It's the kubernetes client API function call (the parameter type needs to be `callable`).  I'll update this to make it more clear.